Library/Homebrew/cmd/desc.rb

```diff
@@ -35,7 +35,7 @@ def desc_args
 
       conflicts "--search", "--name", "--description"
 
-      named_args :formula
+      named_args :formula, min: 1
```

Comment:
I intentionally didn't add a minimum here because of the error message text. With the minimum, here are some sample cases:

```console
$ brew desc
Usage: brew desc [options] text|/text/|formula [text|/text/|formula
...]

...

Error: Invalid usage: this command requires a formula argument

$ brew desc --search
Usage: brew desc [options] text|/text/|formula [text|/text/|formula
...]

...

Error: Invalid usage: this command requires a formula argument
```

This is fine, but in the second case, the missing argument isn't really a formulae argument. Is there a specific reason you changed? I'm happy to keep it as-is if so, but otherwise, I think I'd rather it be changed bask (which I will gladly do)